.fixtures.yml

```diff
@@ -1,4 +1,8 @@
 fixtures:
+  forge_modules:
+    stdlib:
+      repo: puppetlabs/stdlib
+      ref: 8.0.0
```

Comment:
for unit tests, we want to test against the latest code from our dependencies. That's why we specify them as git repos, not forge releases. Please update it like it is below for yumrepo. Don't set the puppet_version parameter.